app/models/supplejack_api/schema_definition.rb

```diff
@@ -36,8 +36,9 @@ def field(type, name, options = {}, &block)
         field = Field.new(name, options, &block)
         self.fields[name] = field
 
-        # This is so we have a string alternative of an integer field that can be used for facetting.
-        field(:string, "#{name}_str", options.merge(type: :string), &block) if type == :integer
+        # This is so we have a string alternative of an integer and date fields that can be used for facetting.
```

Comment:
Is there another type we should take into account?

Reply:
Fair question! Iv had a look through the whole schema and we only have Point fields for Date and Integer. The other fields that we can define string, boolean, latlon, and text aren't Point fields.